tests/robustness/coverage/coverage_test.go

```diff
@@ -274,25 +325,42 @@ func extractPattern(trace *tracev1.ResourceSpans, key string) (string, bool) {
 	if strings.HasSuffix(k, "/") {
 		suffix = "/"
 	}
-	switch strings.Count(strings.TrimRight(k, "/"), "/") {
+	k = strings.TrimRight(k, "/")
+
+	if strings.HasPrefix(k, "/registry/masterleases") {
+		switch strings.Count(k, "/") {
+		case 2:
+			return "/registry/masterleases" + suffix, true
+		case 3:
+			return "/registry/masterleases/{name}" + suffix, true
+		}
+		return k, false
+	}
+
+	resource := "{resource}"
+	if strings.HasPrefix(k, "/registry/events") {
```

Comment:
Can you leave a comment that `events` are special as that's the only resource that has watch cache disabled by default?

Reply:
I will move it to a separate PR as there are many special cases to handle